base/commandLine.cpp

```diff
@@ -822,6 +823,9 @@ Common::String parseCommandLine(Common::StringMap &settings, int argc, const cha
 			DO_LONG_OPTION_BOOL("disable-sdl-parachute")
 			END_OPTION
 
+			DO_LONG_OPTION_BOOL("disable-sdl-audio")
```

Comment:
We need to add it to the help screen, then. Also, since it is SDL-specific, shouldn't it be guarded around by `#ifdef SDL_BACKEND`?